dotnetv3/Kinesis/README.md

```diff
@@ -0,0 +1,72 @@
+# Amazon Kinesis code examples in C#
+
+## Purpose
+
+This folder contains examples that show how to use the AWS SDK for .NET 3.x to
```

Comment:
Kinesis -> Amazon Kinesis 

(title doesn't count as first use, right?)

You could also subequently just use 'Kinesis'

Reply:
I've changed so that the title says "Amazon Kinesis" and the first reference under purpose also does. Then the rest of the references are just "Kinesis"